requirements.txt

```diff
@@ -3,5 +3,5 @@ sip==4.19.7
 requests>=2.22.0
 psutil>=5.4.7
 behave>=1.2.6
-fbs==0.8.3
+fbs==0.6.6
```

Comment:
we're back?  Did this get reverted on accident?

Reply:
Yes, I reverted it yesterday. As you mentioned, I though 0.6.6 is the version run well on all of our testing machines. I can change it back if I fixed it wrong.

Reply:
TL;DR: `requirements.txt` should be deprecated in favor of `Pipfile` and `Pipfile.lock` per #64.

We had been using `fbs 0.6.6` because it was one of the last versions of `fbs` that did _not_ require us to refactor the directory structure of the application (i.e. add `src/main/python`). Because we've now gone through the trouble, I think it's best that we use the latest version of `fbs` and continually update the application to work with it's, hopefully improving, API. I also think it's a good idea that we deprecate `requirements.txt` in favor of `Pipfile` (via [pipenv](https://docs.pipenv.org/en/latest/)) as it provides some pretty cool features for locking dependencies at certain versions as well as across platforms. Basically, I think it will help solve a lot of the dependency issues we've had with using `fbs`. As an aside, I think the Python community at large is also moving towards using it as the de-facto dependency management tool and I've found it much more enjoyable to use over `pip` and `virtualenv`.

Reply:
@samgomena solid point. Let's update the fbs to the last version. By the way, kinda embarrassed, it takes me some time to update my workspace to be ablt to use `pipenv`. Our `Pipfile` restrict it to be on `python 3.6` but my machine is on `python 3.7.4`. Let's keep `requirements.txt` for a while. I will take a time to update my machine.